mqboost/objective.py

```diff
@@ -103,16 +127,20 @@ def _compute_grads_hess(
     _y_train, _y_pred = _train_pred_reshape(
         y_pred=y_pred, dtrain=dtrain, len_alpha=_len_alpha
     )
-    grads = []
+    grads = []; hess = []
```

Comment:
```suggestion
    grads = []
    hess = []
```
I personally prefer this. Can you change this?